planning/scenario_planning/scenario_selector/package.xml

```diff
@@ -20,6 +20,9 @@
   <exec_depend>topic_tools</exec_depend>
   <exec_depend>ros2cli</exec_depend>
 
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
```

Comment:
Like in #154 we could change this to just include `ament_cmake_cppcheck`

Reply:
Fixed in a8cf7429